test/units/testsuite-50.sh

```diff
@@ -187,6 +187,7 @@ if [ "${HAVE_OPENSSL}" -eq 1 ]; then
     sfdisk --part-label "${image}.gpt" 3 "Signature Partition"
 fi
 loop="$(losetup --show -P -f "${image}.gpt")"
+udevadm wait --timeout 60 --settle "${loop:?}"
```

Comment:
shouldn't this be `udevadm wait --timeout 60 --settle "${loop}p1" "${loop}p2"

Reply:
According to @yuwata's finding that causes issues - see https://github.com/systemd/systemd/commit/1af39ef2f6ea7f92ea2cc35b3a5d0e720db6ea4a